packages/calcite-design-tokens/package.json

```diff
@@ -9,31 +9,10 @@
     "tokens"
   ],
   "type": "module",
-  "main": "./dist/js/global.js",
-  "types": "./dist/js/global.d.ts",
+  "main": "./dist/es6/global.js",
```

Comment:
Does this need to be reverted since we're keeping the JS output?

Reply:
Makes sense, but this is now unrelated to this PR and I _think_ falls under breaking change territory.